Spec/Auth.swift

```diff
@@ -1717,14 +1718,12 @@ class Auth : QuickSpec {
                     waitUntil(timeout: testTimeout) { done in
                         auth.authorise(nil, options: authOptions) { tokenDetails, error in
                             expect(authCallbackHasBeenInvoked).to(beTrue())
-
-                            authCallbackHasBeenInvoked = false
```

Comment:
> Not sure what you mean by target

Well, Xcode project can contain several build targets - e.g.. main iOS or OSX app, unit tests, apple watch app, app extension, framework, etc. 

In our case we have `Ably` ObjC library target, `AblySpecs` Swift tests target, and `AblyTests` ObjC tests target. 

So my question was, in another words, - should I remove use of `force` from tests only, or should I remove `force` property from `AuthOptions` class in `Ably` library implementation too? 

I guess I just need to remove it from test target only, otherwise it won't be possible to implement `RSA10d` and `RTC8`, because both of them needs `force`.

 Am I right here, could you confirm please @mattheworiordan ?